src/xenia/kernel/xboxkrnl/xboxkrnl_strings.cc

```diff
@@ -334,7 +337,8 @@ int32_t format_core(PPCContext* ppc_context, FormatData& data, ArgList& args,
               // functions and with C in wprintf functions."
               is_wide = false;
             } else {
-              is_wide = ((flags & FF_IsWide) != 0) ^ wide;
```

Comment:
You can undo this change, and change the `if (flags & FF_IsLong) {` just above to `if (flags & (FF_IsLong | FF_IsWide)) {`.